docs/api/wooga/gradle/unity/APICompatibilityLevel.html

```diff
@@ -29,7 +29,7 @@
     <!-- Generated by groovydoc -->
     <title>APICompatibilityLevel (Gradle Unity plugin API)</title>
     
-    <meta http-equiv="Content-Type" content="text/html; charset=UTF-8">
+    <meta http-equiv="Content-Type" content="text/html; charset=windows-1252">
```

Comment:
Ok better you don't commit the files from windows ;)

Reply:
Yes :(